docs/unpackaged.rst

```diff
@@ -1,14 +1,212 @@
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+        <compactLayouts>
+            <fullName>NPSP_Household_Account</fullName>
+            <fields>Name</fields>
+            <fields>npo02__TotalOppAmount__c</fields>
+            <fields>%%%NAMESPACE%%%Number_of_Household_Members__c</fields>
+            <label>NPSP Household Account</label>
+        </compactLayouts>
+    </CustomObject>
+
+..
+
+    Note that only the reference to the NPSP field ``Number_of_Household_Members__c`` is tokenized. (This field is called ``npsp__Number_of_Household_Members__c`` when installed as part of the managed package.) References to NPSP's own managed package dependency, ``npo02``, are not tokenized because this metadata is always namespaced when installed.
+
+    If this metadata isn't tokenized, it fails to deploy into an org containing NPSP as a beta or released managed package.
+
+.. note:: The resolution of component references in namespaced scratch orgs and in managed installations of the same metadata are not identical. Metadata that is tokenized and deploys cleanly in a namespaced scratch org can still fail in a managed context.
+
+
+
+Retrieve Unpackaged Metadata
+----------------------------
+
+CumulusCI provides tasks to retrieve changes to unpackaged metadata, just as with packaged metadata. For more details on these tasks, see `the dev section<TODO>`_.
+
+When working with unpackaged metadata, it's important to maintain awareness of key considerations related to retrieving metadata that is not part of the main application.
+
+* Take care to separate your development between the different bundles you wish to retrieve.
+    Example: If you have changes to make in the application as well as in unpackaged metadata, complete the application changes first, retrieve them, and then make the unpackaged changes and retrieve those. If you conflate changes to components that live in separate elements of your project, it's difficult to untangle them.
+* Whenever possible, build your unpackaged metadata in an org that contains a beta or released managed package. By doing so, the metadata contains namespaces when extracted, which CumulusCI easily replaces with tokens when retrieving metadata. It's difficult to manually tokenize metadata that's retrieved from an unmanaged org without namespaces. 
+
+After building changes to unpackaged metadata in a managed org, retrieve it using ``retrieve_changes`` with the additional ``namespace_tokenize`` option, and use the ``path`` option to direct the retrieved metadata to your desired unpackaged directory.
+
+    Example: Run ``retrieve_changes`` to retrieve metadata changes into the ``unpackaged/config/qa`` subdirectory, and replace references to the namespace ``npsp`` with the appropriate token.
+
+.. code-block:: console
+
+    $ cci task run retrieve_changes --path unpackaged/config/qa --namespace_tokenize npsp
```

Comment:
Swapped out the old -o flags. Please confirm this updated command is correct.

Reply:
This looks correct to me.

Reply:
10-4